<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtBinaryTreeDemoClass</class>
 <widget class="QMainWindow" name="QtBinaryTreeDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1620</width>
    <height>890</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1620</width>
    <height>890</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1620</width>
    <height>890</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtBinaryTreeDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="AddElementButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj Wartość</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteElementButton">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń Wartość</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ObjectValueEditBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>460</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="ObjectNameEditBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Wartość</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwa Obiektu</string>
    </property>
   </widget>
   <widget class="RenderArea" name="TreeDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>59</y>
      <width>1600</width>
      <height>781</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1620</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNowe_Drzewo">
    <property name="title">
     <string>Nowe Drzewo</string>
    </property>
    <addaction name="actionInteger"/>
    <addaction name="actionFloat"/>
    <addaction name="actionString"/>
   </widget>
   <addaction name="menuNowe_Drzewo"/>
  </widget>
  <action name="actionInteger">
   <property name="text">
    <string>Integer</string>
   </property>
  </action>
  <action name="actionFloat">
   <property name="text">
    <string>Float</string>
   </property>
  </action>
  <action name="actionString">
   <property name="text">
    <string>String</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header location="global">PaintArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtBinaryTreeDemo.qrc"/>
 </resources>
 <connections/>
</ui>
